app/mailers/welcome_mailer.rb

```diff
@@ -1,7 +1,7 @@
 class WelcomeMailer < ApplicationMailer
   def welcome_notification(user)
     mail(
-      to: email_with_name(user.email_address, user.full_name),
+      to: email_to_user(user),
```

Comment:
I consider it friendlier to use the recipient's name, so just a UX benefit. Same with `from`.